.github/workflows/semantic-pr.yml

```diff
@@ -2,7 +2,7 @@
 name: "Semantic PR"
 
 on:  # yamllint disable-line rule:truthy
-  pull_request_target:
```

Comment:
Why change to just pull_request? Easier to test but easier to skirt around (?)

Reply:
it should have been pull_request from the beginning. As we are using the workflow workarround to get write permission